extensions/vertx-http/deployment/src/test/resources/conf/http-roles-grant-config.properties

```diff
@@ -0,0 +1,15 @@
+quarkus.http.auth.basic=true
+quarkus.http.auth.policy.t1.roles.admin=Admin1
```

Comment:
Can you please add one more test where  `roles-allowed=Admin1` etc is set in the policy, without annotations

Reply:
> Can you please add one more test where roles-allowed=Admin1 etc is set in the policy, without annotations

I'm not sure I understand.

1. yes, I'll add more tests
2. you can't really expect that you roles mapped via path matching HTTP security policy can also be checked by path matching HTTP security policy. That's chicken egg problem, because roles are only assigned when policy is matched and complied (e.g. if you have roles required by the role-based policy, then and only then you can get mapped roles). I'll make it clear in docs that you are expect to use these new roles with `@RolesAllowed` annotation.

Reply:
Okay, so I've added new tests and covered what I think is relevant. This works the same way granting permissions work.

Reply:
Please see `quarkus.http.auth.policy.t6.roles-allowed=Admin1`.

Reply:
Done.